engine/src/utils/mod.rs

```diff
@@ -159,45 +161,53 @@ impl fmt::Display for TextFilter {
     }
 }
 
-pub fn db_path() -> Result<&'static str, Box<dyn std::error::Error>> {
-    if let Some(path) = ARGS.db.clone() {
-        let mut absolute_path = if path.is_absolute() {
-            path
-        } else {
-            env::current_dir()?.join(path)
+pub static DB_PATH: LazyLock<Result<Cow<'static, Path>, sqlx::Error>> = LazyLock::new(|| {
```

Comment:
I'm a bit between two chairs. maybe a &str is better then a path?

Reply:
Inside ffplayout it is always used as a `Path`. Sqlx allows passing arguments, but even that would be allowed in a `Path`.
But most important, making it a `&str` would require the `PathBuf` returned by `env::current_dir()` to be converted using `Path::to_string_lossy`, which just feels wrong.